operator/validator/controller.go

```diff
@@ -1250,7 +1214,11 @@ func SetupCommitteeRunners(
 	return func(slot phase0.Slot, shares map[phase0.ValidatorIndex]*spectypes.Share) *runner.CommitteeRunner {
 		// Create a committee runner.
 		epoch := options.NetworkConfig.Beacon.GetBeaconNetwork().EstimatedEpochAtSlot(slot)
-		valCheck := TempBeaconVoteValueCheckF(options.Signer, slot, options.SSVShare.Share.SharePubKey, epoch) // TODO: (Alan) fix slashing check (committee is not 1 pubkey)
+		sharepbs := make([]spectypes.ShareValidatorPK, len(shares))
+		for i, s := range shares {
+			sharepbs[i] = s.SharePubKey
+		}
+		valCheck := specssv.BeaconVoteValueCheckF(options.Signer, slot, sharepbs, epoch)
```

Comment:
It is best to pass only shares of validators doing the attestation duty...
since only they are slashable

Reply:
Done, please check updated code.